Swift/Database.swift

```diff
@@ -220,6 +220,27 @@ public final class Database {
     }
     
     
+    /// Sets an expiration date on a document.
+    /// After this time the document will be purged from the database.
+    ///
+    /// - Parameters:
+    ///     - id: The ID of the document to set the expiration date for
+    ///     - date: The expiration date. Set nil date will reset the document expiration.
+    /// - Throws: An error on a failure.
+    public func setDocumentExpiration(id: String, date: Date?) throws {
```

Comment:
let's do `setDocumentExpiration(withID documentID: String, expiration: Date?) throws` to be consistent with purgeDocument(withID).